changelog.txt

```diff
@@ -2,6 +2,90 @@
+* Embeds: update Loom logo.
+* External Media: add feature to the Cover block.
+* Google Calendar Block: improve the display of the block in the editor.
+* Image Compare Block: accessibility improvements.
+* Instant Search: improve accessibility of the Search modal.
+* Likes: remove jQuery dependency where possible.
+* Pay with PayPal Block: improve the styling of the buttons in subscription emails.
+* Pay with Paypal Block: improve the display of currencies.
+* Podcast Player Block: add option to hide the episode title.
+* Podcast Player Block: display more helpful error messages when a podcast cannot be embedded.
+* Publicize: improve the usability of the Twitter thread options.
+* REST API: support needed capabilities in Jetpack REST API endpoints that allow site based authentication.
+* Social Menu & Social Media Icons: add support for Telegram profiles.
+* Synchronization: improve reliability of synchronization of theme changes.
+* WhatsApp Block: add option to change the alignment of the button.
+* Widget Visibility: improve performance on sites with a large number of pages.
+* WordPress.com Block Editor: add option to add metadata to a post indicating the last editor used.
+* WordPress.com REST API: remove outdated code that allowed non-secure requests to the API, as such requests are no longer accepted.
+
+**Improved compatibility**
+
+* Comments: avoid AMP validation errors when using Jetpack's Comments feature and the AMP plugin.
+* Dashboard: better support all states available when using one of Jetpack's Security solutions.
+* Embeds: improve compatibility between Jetpack's Instagram embed and WordPress' own embed.
+* Embeds: ensure Facebook Embeds work well with the AMP plugin.
+* External Media: update iconography and better forward compatibility.
+* General: better detect local development environments.
+* Google Analytics: add support for the AMP plugin to track WooCommerce events.
+* OpenTable Block: ensure full compatibility with the AMP plugin.
+* Pay with PayPal Block: ensure full compatibility with the AMP plugin.
+* Sharing: add support for the Google Web Stories plugin.
+* Widget Visibility: avoid performance impact it may have on the block editor when used with the Gutenber plugin.
+* YouTube embeds: ensure that all YouTube video embeds work well when using the AMP plugin.
+
+**Bug fixes**
+
+* Calendly and Eventbrite Blocks: fix layout issue when selecting block styles in the editor.
+* Contact Form: selection widgets, radio buttons, and checkboxes can now use commas, brackets, and backslashes in the labels and values without breaking the form.
+* Donations / Payments Block: fix visual bug in Stripe connection banner.
+* Embeds: ensure Facebook videos are centered properly with the Twenty Twenty theme.
+* Image CDN: the CDN will now ignore attempts to specify percentages for width or height in an image tag.
+* Infinite Scroll: ensure the number of posts loaded when using Infinite Scroll respect Posts per page settings.

```

Comment:
"ensure the number [...] respects"
```suggestion
* Infinite Scroll: ensure the number of posts loaded when using Infinite Scroll respects Posts per page settings.
```
I'm also unsure about the capitalization of "Posts per page" there. Should it be "Posts per Page" or "posts per page" instead?

Reply:
Good point. 974f2464b4eb3205e1153c6469b8a766d9ac14a8